helloworld/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "helloworld",
+  "version": "1.0.0",
+  "description": "node code for lab",
+  "main": "hello.js",
+  "scripts": {
+    "lint": "./node_modules/eslint/bin/eslint.js",
+    "test": "./node_modules/mocha/bin/mocha"
```

Comment:
Good job remembering to add scripts. The lint script is missing a `.` after it. Without the dot, "npm run lint" doesn't work properly when I type it in terminal. So your lint script should look like: 

```
"lint": "./node_modules/eslint/bin/eslint.js ."

```